### PR TITLE
style url-generator

### PR DESCRIPTION
**Pull request by sdobbert**
Follow up: Styling for the KLIPS URL-Generator.

image

### CODE REVIEW

File: easy-to-use-api/klips-url-generator/src/App.tsx [hunk 1]
```diff
@@ -82,9 +84,11 @@ const App: React.FC = () => {
 
   const changeRegion = (newRegion: string) => {
     setRegion(newRegion);
+    setSelectStatusRegion("");
   };
 
   const onDrawEnd = (geom: OlGeometry) => {
+
```
Comment (FritzHoing) [inline easy-to-use-api/klips-url-generator/src/App.tsx:91]:
```suggestion
```

File: easy-to-use-api/klips-url-generator/src/components/DrawGeometry/index.tsx [hunk 2]
```diff
@@ -54,7 +54,7 @@ interface GetCoordinatesStringProps extends ButtonGroupProps {
 /**
  * The Button.
  */
-const GetCoordinatesString: React.FC<GetCoordinatesStringProps> = ({
+const DrawGeometry: React.FC<GetCoordinatesStringProps> = ({
```
Comment (FritzHoing) [inline easy-to-use-api/klips-url-generator/src/components/DrawGeometry/index.tsx:57]: Good hint! I thought we would have changed that name already.

File: easy-to-use-api/klips-url-generator/src/App.tsx [hunk 3]
```diff
@@ -108,51 +113,61 @@ const App: React.FC = () => {
...
-      <MapContext.Provider value={map}>
-        <div className='output-wrapper'>
-          <div className='header'>
-            <img src="https://www.klips-projekt.de/wp-content/uploads/2021/02/SAG_KLIPS-Logo_Jan21.png" alt="KLIPS Logo"></img>
-            <h2>Widget URL Generator</h2>
+      <ConfigProvider
+        theme={{
+          algorithm: theme.defaultAlgorithm,
+          token: {
+            colorPrimary: '#e45f24',
```
Comment (FritzHoing) [inline easy-to-use-api/klips-url-generator/src/App.tsx:139]: Could that me moved to the constants file?

**Comment by FritzHoing** [comment]
Very nice @sdobbert !

File: easy-to-use-api/klips-url-generator/src/App.tsx (hunk 3 above)
Comment (sdobbert) [reply easy-to-use-api/klips-url-generator/src/App.tsx]: Thank you, I moved the token to the constants file.

**Comment by hblitza** [comment]
Will be closed in favor of #228

**Pull request closed without merging**